reactive/calico.py

```diff
@@ -77,6 +77,11 @@ def upgrade_charm():
         })
 
 
+@hook('pre-series-upgrade')
+def pre_series_upgrade():
+    status_set('blocked', 'Series upgrade in progress')
```

Comment:
It seems like this status could easily get overwritten by other reactive handlers that run during the pre-series-upgrade hook.